### DISCUSSION TITLE
Missing information from documentation

### DISCUSSION THREAD

**Discussion by jrgoncal** [FirstGearGames/FishNet#302] [Documentation]
There are a few areas I've noticed that are missing short but vital information, which could lead to people wondering why things aren't working. I can't remember them all now, but the two I've been stuck on lately are:

**Synchronize Parent**
https://fish-networking.gitbook.io/docs/master/pro#features-and-enhancements
_All instances of Synchronize Parent (including hover-comment in editor) should mention that the object requires a NetworkBehavior (with EmptyNetworkBehavior as an option if no others are being used)._

**Channel.Unreliable**
https://firstgeargames.com/FishNet/api/api/FishNet.Transporting.Channel.html
https://fish-networking.gitbook.io/docs/manual/guides/remote-procedure-calls
_There are numerous examples of how to use the channels, but no where describes what exactly Unreliable does (or doesn't do). Does it send data out of order? Chance to drop data completely? I'm using it for healthbars and have no idea if that's a good idea or not lol._

**Comment by FirstGearGames** [comment]
I agree on both of these accounts. I'll have to put reliable/unreliable in the terminology section. Thank you.

**Comment by FirstGearGames** [comment]
The synchronize parent is going away in favor of a better approach in v4.
I've provided documentation in Terminology > Communicating guide on Channel and Eventual Consistency.
